djangoapp/djangoapp/.gitattributes

```diff
@@ -0,0 +1 @@
+settings.py filter=git-crypt diff=git-crypt
```

Comment:
I don't think we should encrypt `settings.py` entirely, as most of the contents aren't sensitive. Sensitive values and values that vary across environments should be specified as environment variables, and read into `settings.py` using `os.environ.get()`.

Reply:
removed